parseSCSS.js

```diff
@@ -12,21 +12,32 @@ const parser = (css) => {
   */
   const brandParse = postcss.plugin('brand-parse', (options) => {
     return (global, result) => {
-      global.walkRules(rule => {
-        rule.walkAtRules(at => {
-          if (/brand/.test(at.params)) {
-            splits.push(`${rule.selector} {`);
+      global.walkAtRules(at => {
```

Comment:
I think it would help readability to pull these two walk loops out into their own functions.